### DISCUSSION TITLE
Description text files

### DISCUSSION THREAD

**Discussion by LoveNug** [anapnoe/stable-diffusion-webui-ux#106] [Q&A] [answered]
I'm not sure whether this has been covered or not, but is there any chance the display of .description.txt file contents in the checkpoint/lora/embedding cards will be enabled? I generally store useful info there like triggers, VAE preferences, etc. It's a handy function to have, which I enjoy in Auto1111. I thoroughly prefer this fork though!

**Comment by anapnoe** [comment, marked as answer]
sorry it is hidden by default want to add inline editing capabilities and find a way to display arbitrary number o text probably with a toggle tooltip you can create a request for this
